### PR TITLE
allow single address wallet for auditable reasons

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fix : #4225

### CODE REVIEW

File: public/views/create.html [hunk 1]
```diff
@@ -167,7 +167,13 @@
             <div class="oh"  ng-show="create.seedSourceId == 'new'">
               <label for="network-name" >
                 <span>Testnet</span>
-                <switch id="network-name" name="isTestnet" ng-model="isTestnet" class="green right m5t m10b"></switch>
+                <switch id="network-name" name="isTestnet" ng-model="isTestnet" class="green right m5t m15b"></switch>
+              </label>
+            </div>
+            <div class="oh">
+              <label for="single-address">
+                <span translate>Single Wallet Address</span> <small translate>For auditable reasons</small>
```
Comment (isocolsky) [inline public/views/create.html]: I think "for audit purposes" is a better description.

File: test/controllers/create.test.js [hunk 2]
```diff
@@ -122,6 +122,7 @@ describe('createController', function() {
     scope.walletName = 'A test wallet';
     scope.isTestnet = false;
     scope.bwsurl = null;
+    scope.isSingleAddress = false;
```
Comment (isocolsky) [inline test/controllers/create.test.js]: Where do you set `isSingleAddress` to the actual value?

**Comment by JDonadio** [comment]
After ionic integration, you should change **switch** tag fot **ion-toggle**

File: src/js/controllers/index.js [hunk 3]
```diff
@@ -304,6 +304,9 @@ angular.module('copayApp.controllers').controller('indexController', function($r
           if (err) {
             self.updateError = bwsError.msg(err, gettext('Could not update Wallet'));
           } else {
+            fc.singleAddress = !!ret.singleAddress;
+            console.log("################");
+            console.log(fc.singleAddress);
```
Comment (cmgustavo) [inline src/js/controllers/index.js]: Please, remove console.loc

**Comment by gabrielbazan7** [comment]
Updated

File: public/views/create.html [hunk 4]
```diff
@@ -159,6 +159,13 @@
                 </ion-toggle>
               </label>
             </div>
+            <div class="oh" ng-show="create.seedSourceId == 'new'">
+              <label for="single-address" class="dbi">
+                <span translate>Single Wallet Address</span> <small translate>For audit purposes</small>
```
Comment (isocolsky) [inline public/views/create.html]: Should be "Single address wallet"

**Comment by gabrielbazan7** [comment]
Updated

**Comment by isocolsky** [comment]
ACK

**Pull request merged**